### PR TITLE
stop building ghc810

### PR DESCRIPTION
**Pull request by MangoIV**
as we remove ghc8107 support I removed it from the checks, the apps and the devShells

### CODE REVIEW

**Comment by MangoIV** [comment]
I have not removed all the code to make it easier to revive it if we actually have to repair the support at some point, I hope that makes sense, @L-as ?

**Comment by L-as** [comment]
It's blocking CI

---元のメッセージ---
　 2022年7月4日 14:11, Mango The Fourthに が送信

File: flake.nix [hunk 1]
```diff
@@ -555,8 +551,8 @@
 
       devShells = perSystem (system:
         {
-          "ghc810" = self.ghc810Flake.${system}.devShell;
           "ghc9" = self.flake.${system}.devShell;
+          "ghc810" = self.ghc810Flake.${system}.devShell;
```
Comment (L-as) [inline flake.nix:555]: This should be removed

**Review by L-as — CHANGES_REQUESTED** [review]
Otherwise it will be built by CI

File: flake.nix (hunk 1 above)
Comment (MangoIV) [reply flake.nix:555]: I just thought it doesn't really matter as there's nothing in there really that has to be built extra. (Hls comes from nixpkgs and is reliably cached) but if you wish I'll remove it. ;)

Comment (L-as) [reply flake.nix:555]: A whole extra GHC potentially, and all the libraries we depend on. That's a lot.

**Pull request merged**